unified-installer/astra-unified-installer.sh

```diff
@@ -2793,62 +2793,65 @@ if trident_will_be_installed_or_modified; then
+                    _warning_message+=" ($(get_config_trident_image). Astra Control Provisioner (Trident ACP)"
+                    _warning_message+=" support (23.10+) is not guaranteed."
                     logwarn "$_warning_message"
 
                     step_generate_torc_patch "${_generate_torc_args[@]}"
                     trident_operator_image_needs_upgraded && step_generate_trident_operator_patch
-                elif prompt_user_yes_no "Would you like to upgrade Trident?"; then
+                elif prompt_user_yes_no "Would you like to upgrade Astra Trident?"; then
                     step_generate_torc_patch "${_generate_torc_args[@]}"
                     trident_operator_image_needs_upgraded && step_generate_trident_operator_patch
                 else
-                    _msg="You have chosen to use a version of Trident that is not supported with the current version"
+                    _msg="You have chosen to use a version of Astra Trident that is not supported with the current version"
                     _msg+=" of Astra Control. This may result in some App Data Management operations not functioning"
                     _msg+=" correctly or being blocked within Astra Control. It is highly recommended to upgrade"
-                    _msg+=" Trident to ensure compatibility and proper functionality."
+                    _msg+=" Astra Trident to ensure compatibility and proper functionality."
                     logwarn "$_msg"
                 fi
             # Trident operator upgrade (standalone)
             elif trident_operator_image_needs_upgraded; then
-                if config_trident_operator_image_is_custom || prompt_user_yes_no "Would you like to upgrade the Trident Operator?"; then
+                if config_trident_operator_image_is_custom || prompt_user_yes_no "Would you like to upgrade the Astra Trident Operator?"; then
                     step_generate_trident_operator_patch
                 else
-                    loginfo "Trident Operator will not be upgraded."
+                    loginfo "Astra Trident Operator will not be upgraded."
                 fi
             fi
         else
-            logdebug "Skipping Trident upgrade (COMPONENTS=${COMPONENTS}, DO_NOT_MODIFY_EXISTING_TRIDENT=${DO_NOT_MODIFY_EXISTING_TRIDENT})"
+            logdebug "Skipping Astra Trident upgrade (COMPONENTS=${COMPONENTS}, DO_NOT_MODIFY_EXISTING_TRIDENT=${DO_NOT_MODIFY_EXISTING_TRIDENT})"
         fi
 
         # Upgrade/Enable ACP?
         if components_include_acp; then
             # Enable ACP if needed (includes ACP upgrade)
             if ! acp_is_enabled; then
-                if config_acp_image_is_custom || prompt_user_yes_no "Would you like to enable ACP?"; then
+                if config_acp_image_is_custom || prompt_user_yes_no "Would you like to enable the Astra Control Provisioner (Trident ACP)?"; then
```

Comment:
Here, I used `Astra Control Provisioner`, but added `(Trident ACP)` to call back to the image name and the fact that it is part of Trident. Similar problem as before though as it's kind of a lot of fluff, so not sure what to do.

Reply:
If we want to stay consistent with the docs, this should just be "Would you like to enable Astra Control Provisioner?"